rero_ils/modules/operation_logs/serializers.py

```diff
@@ -27,6 +27,10 @@ class OperationLogsJSONSerializer(JSONSerializer, CachedDataSerializerMixin):
 
     def _postprocess_search_hit(self, hit: dict) -> None:
         """Post-process each hit of a search result."""
+        # add library name if the library entry exists.
+        if library := hit.get('metadata', {}).get('library'):
+            if lib := Library.get_record_by_pid(library.get('value')):
+                library['name'] = lib.get('name')
```

Comment:
you should use "CachedDataSerializerMixin" behavior : 
`library['name'] = self.get_resource(LibrariesSearch(), pid)['name']`